### ISSUE TITLE
Add pie chart for analytics

### ISSUE THREAD

**Issue by sergio222-dev** [sergio222-dev/ztools-2#81]
**give** the need of show data
**when** the user click in the Reports button
**then** show a chart like the mentioned below

using this library https://www.amcharts.com/demos/#pie

**Comment by sergio222-dev** [comment]
payload expected in frontend

```json
{
  "data": [
    {
          "categoryName": "Salamin",
         "totalInflow": "2500000",
         "totalOutflow": "500000"
     },
    {
          "categoryName": "Salamin2",
         "totalInflow": "2500000",
         "totalOutflow": "500000"
     },
    {
          "categoryName": "Salamin3",
         "totalInflow": "2500000",
         "totalOutflow": "500000"
     }
  ]
}
```

**Comment by sergio222-dev** [comment]
resolved by #109
